build.gradle

```diff
@@ -40,7 +40,7 @@ dependencies {
     ]
 
     // Used for compiling the main classes
-    groovyClasspath 'org.codehaus.groovy:groovy-all:2.5.12'
+    groovyClasspath 'org.codehaus.groovy:groovy-all:3.0.10'
```

Comment:
@runningcode @ghale Why do we need to explicitly add the groovy dependency? Isn't it implicitly added by the Groovy plugin?

Reply:
Good question. This was here before because of this comment: https://github.com/gradle/android-cache-fix-gradle-plugin/pull/261/files#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7L57

Now that we're upgraded to Groovy 3, it is no longer necessary. See this PR : https://github.com/gradle/android-cache-fix-gradle-plugin/pull/261